CHANGELOG.md

```diff
@@ -1,102 +1,65 @@
-## Forest unreleased
+## Forest v0.7.0 (2023-03-23)
```

Comment:
Do we need to change `workspace.package.version` in `Cargo.toml` as well?

Reply:
yes did